src/lib/code-sample/ruby.ts

```diff
@@ -0,0 +1,42 @@
+import { pascalCase, snakeCase } from 'change-case'
+
+import type { CodeSampleDefinition, Context } from './schema.js'
+
+export const createRubyRequest = (
+  { request }: CodeSampleDefinition,
+  _context: Context,
+): string => {
+  const parts = request.path.split('/')
+  const params = Object.entries(request.parameters)
+    .map(([key, value]) => `${snakeCase(key)}: ${JSON.stringify(value)}`)
+    .join(', ')
+
+  return `seam${parts.map((p) => snakeCase(p)).join('.')}(${params})`
+}
+
+export const createRubyResponse = (
+  { response, title }: CodeSampleDefinition,
+  context: Context,
+): string => {
+  const { endpoint } = context
+
+  if (endpoint.response.responseType === 'void') return 'nil'
+
+  const { responseKey } = endpoint.response
+  const responseValue = response?.body?.[responseKey]
+
+  if (responseValue == null) {
+    throw new Error(`Missing ${responseKey} for '${title}'`)
+  }
+
+  const responseRubyClassName = pascalCase(responseKey)
+  const responseRubyParams = Object.entries(responseValue)
+    .map(([paramKey, paramValue]) => {
+      const formattedValue =
+        paramValue === null ? 'nil' : JSON.stringify(paramValue)
+      return `  ${snakeCase(paramKey)}=${formattedValue}`
+    })
+    .join('\n')
+
+  return `<Seam::${responseRubyClassName}:0x00438\n${responseRubyParams}>`
```

Comment:
Where does `0x00438` come from?

Reply:
The `0x00438` is the unique memory address of the object, which is an instance of the Seam class. 
Do we need to randomize it each time? For a sample, I thought the static address is sufficient, but I'm open to generating it.

Reply:
Sure. Maybe just set it to something less random, like 0x00001 or 0x00000 just so it appears insignificant

Reply:
95929253b3d5ed66ce272e50325c72a7a8bc11a5